core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -73,6 +74,13 @@ public static void updateWidget(Context context, WidgetState widgetState) {
         } else {
             startMediaPlayer = new MainActivityStarter(context).withOpenPlayer().getPendingIntent();
         }
+
+        PendingIntent startPlaybackSpeedDialog;
+        if (widgetState.media != null && widgetState.status == PlayerStatus.PLAYING) {
+            startPlaybackSpeedDialog = new PlaybackSpeedActivityStarter(context).getPendingIntent();
+        } else {
+            startPlaybackSpeedDialog = new MainActivityStarter(context).withOpenPlayer().getPendingIntent();
+        }
```

Comment:
I think it would be more consistent to always show the playback speed dialog. Otherwise, users might click that button and then wonder why the app opens instead of the dialog.